ansible/playbooks/roles/repository/files/download-requirements/src/mode/red_hat_family_mode.py

```diff
@@ -87,14 +87,26 @@ def _add_third_party_repositories(self):
 
     def __remove_dnf_cache_for_untracked_repos(self):
         # clean metadata for upgrades (when the same package can be downloaded from changed repo)
-        repoinfo: List[str] = self._tools.dnf.list_all_repos_info()
-        repocaches: List[str] = [repodir for repodir in Path('/var/cache/dnf').iterdir() if repodir.is_dir()]
-
-        for repo in repoinfo:
-            if repo['Repo-status'] == 'disabled':
-                for repocache in repocaches:
-                    if repocache.name.startswith(repo['Repo-id']):
+        repocaches: List[str] = list(Path('/var/cache/dnf').iterdir())
+
+        repo_names = [
```

Comment:
```suggestion
        repo_ids = [
```
Example difference (from `dnf repoinfo`):
```
Repo-id            : rhui-rhel-8-for-x86_64-appstream-rhui-rpms
Repo-name          : Red Hat Enterprise Linux 8 for x86_64 - AppStream from RHUI (RPMs)
```